dist/vignette.cjs.js

```diff
@@ -230,7 +230,7 @@ var Vignette = (function () {
         }
         return modeParameters.join('/');
     };
-    Vignette.imagePathRegExp = /\/\/vignette\d?\.wikia/;
+    Vignette.imagePathRegExp = /\/\/vignette(\d|-poz)?\.wikia/;
     Vignette.domainRegExp = /(wikia-dev.com|wikia.nocookie.net)/;
     Vignette.legacyPathRegExp = /(wikia-dev.com|wikia.nocookie.net)\/__cb[\d]+\/.*$/;
```

Comment:
Can we inject this domains as part of the setup? It seems brittle (and very Wikia specific) to have the configuration so tightly coupled here.